packages/twenty-front/src/modules/settings/data-model/fields/forms/relation/components/SettingsDataModelFieldRelationForm.tsx

```diff
@@ -101,7 +101,10 @@ export const SettingsDataModelFieldRelationForm = ({
   });
 
   const selectedObjectMetadataItem = findObjectMetadataItemById(
-    watchFormValue('relation.objectMetadataId'),
+    watchFormValue(
+      'relation.objectMetadataId',
```

Comment:
Is this change intended in this pr @thomtrp ?

Reply:
Yes it fixes the second test